api/schemas/ldapConfiguration.yaml

```diff
@@ -0,0 +1,14 @@
+type: object
+description: LDAP server configuration
+properties:
+  host:
+    type: string
+  username:
+    type: string
+  password:
+    type: string
```

Comment:
E' giusto definire la password come string?

Reply:
Gli unici tipi consentiti da Swagger sono:
string (this includes dates and files), number, integer, boolean, array, object